Specs/DateTime/Spanish/DatePeriodExtractor.json

```diff
@@ -538,28 +412,6 @@
       }
     ]
   },
-  {
-    "Input": "Estare afuera la tercer semana de 2027",
-    "Results": [
-      {
-        "Text": "la tercer semana de 2027",
-        "Type": "daterange",
-        "Start": 14,
-        "Length": 24
-      }
-    ]
-  },
-  {
-    "Input": "Estare afuera la tercer semana proximo año",
```

Comment:
We could change this test to `Estare afuera la tercer semana del proximo año` instead of deleting it.